admin/acceptance.py

```diff
@@ -449,11 +449,14 @@ def main(reactor, args, base_path, top_level):
             make_dispatcher(reactor),
             configure_cluster(control_node=nodes[0], agent_nodes=nodes,
                               certificates=certificates))
+
+        control_node = nodes[0]
```

Comment:
Just above this:

```python
configure_cluster(control_node=nodes[0], agent_nodes=nodes, ...
```

I think that this variable should be moved above that and used there too.